src/pages/networks/actions/DeleteNetworkForwardBtn.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC, useState } from "react";
+import { LxdNetwork, LxdNetworkForward } from "types/network";
+import { queryKeys } from "util/queryKeys";
+import { useQueryClient } from "@tanstack/react-query";
+import {
+  ConfirmationButton,
+  Icon,
+  useNotify,
+} from "@canonical/react-components";
+import { deleteNetworkForward } from "api/network-forwards";
+
+interface Props {
+  network: LxdNetwork;
+  forward: LxdNetworkForward;
+  project: string;
+}
+
+const DeleteNetworkForwardBtn: FC<Props> = ({ network, forward, project }) => {
+  const notify = useNotify();
+  const queryClient = useQueryClient();
+  const [isLoading, setLoading] = useState(false);
+
+  const handleDelete = () => {
+    setLoading(true);
+    deleteNetworkForward(network, forward, project)
+      .then(() => {
+        notify.success(`Network forward for ${forward.listen_address} deleted`);
+        void queryClient.invalidateQueries({
```

Comment:
do we need to invalidate network queries? Maybe just invalidate network forward query for this specific network i.e. `query.queryKey[1] === network.name && query.queryKey[2] === queryKeys.forwards`. I think this might be more efficient when we get back to the `NetworkList` component so that we don't have to query the count for every network all over again.

Reply:
or just use the exact query key for the network forward

Reply:
I'll limit this to the network.

Reply:
Limiting this to the exact forward will not invalidate the forwards list on the network.